ajax/libs/maptalks/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "maptalks",
+  "filename": "maptalks.min.js",
+  "version": "0.37.0",
+  "description": "A light JavaScript library to create integrated 2D/3D maps.",
+  "license": "BSD-3-Clause",
+  "author": "Fu Zhen",
+  "keywords": [
+    "javascript",
+    "gis",
+    "map",
+    "2d-maps",
+    "3d-maps",
+    "geospatial",
+    "maps",
+    "maptalks"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/maptalks/maptalks.js.git"
+  },
+  "homepage": "https://maptalks.org",
+  "npmName": "maptalks",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
```

Comment:
In some very old versions there are Plugins folder and maptalks-demo.js files. I do not sure that we need them.  See v0.2.0. @sufuf3 please help with this issue